ecms_base/features/custom/ecms_projects/config/install/core.base_field_override.taxonomy_term.project_category.path.yml

```diff
@@ -12,7 +12,7 @@ bundle: project_category
 label: 'URL alias'
 description: ''
 required: false
-translatable: false
+translatable: true
```

Comment:
This should remain false

Reply:
Typically we would be using the language prefix (e.g. "/es/") to drive the path, leaving the non-language component matching. My local testing show this working currently, so not sure we want this, but I also don't think it's hurting anything. I'm fine leaving it as is for now